fastlane/Fastfile

```diff
@@ -191,13 +191,13 @@ ENV["PROJECT_ROOT_FOLDER"]="./"
 
     sh("mv ../build/Simplenote.ipa \"../build/Simplenote Internal.ipa\"")
 
-    hockey(
-        api_token: ENV["HOCKEY_API_TOKEN"],        
-        public_identifier: ENV["HOCKEY_PUBLIC_ID"],
-        notify: "0",
-        status: "1",
-        ipa: "./build/Simplenote Internal.ipa",
-        dsym: "./build/Simplenote.app.dSYM.zip",
+    appcenter_upload(
+      api_token: ENV["APPCENTER_API_TOKEN"],
+      owner_name: "Hogwarts-Organization",
+      owner_type: "organization", 
+      app_name: "Simplenote",
+      file: "./build/Simplenote\ Internal.ipa",
```

Comment:
Me again! 🤦‍♂ 

When running this, I got an error saying "Couldn't find build file at path '/Users/jeremymassel/Projects/simplenote-ios/build/Simplenote.ipa'".

It looks like by default, `appcenter_upload` might use the value of `IPA_OUTPUT_PATH`, which in this case doesn't resolve to the new path after the `mv` command above.

In the [example Fastfile](https://github.com/microsoft/fastlane-plugin-appcenter/blob/master/fastlane/Fastfile) for appcenter, I found that they use `file` in just one case, but `ipa` everywhere else  – changing this key to `ipa` made the upload work for me, so I'm wondering if that is "more" correct?

WDYT?

Reply:
mmm... looking at the code `ipa` seems to be deprecated[ here](https://github.com/microsoft/fastlane-plugin-appcenter/blob/0d66522efafad3849e177b475477d30cc5c923e3/lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb#L383), while `file` seems to be the [current way to go ](https://github.com/microsoft/fastlane-plugin-appcenter/blob/0d66522efafad3849e177b475477d30cc5c923e3/lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb#L399).

It seems it performs more checks with the `file` option... let me check if some of them may fail for us for some reason.